bin/MadGraph5_aMCatNLO/cards/production/13TeV/ttznunu_5f_NLO/ttznunu_5f_NLO_run_card.dat

```diff
@@ -0,0 +1,146 @@
+#***********************************************************************
+#                        MadGraph5_aMC@NLO                             *
+#                                                                      *
+#                      run_card.dat aMC@NLO                            *
+#                                                                      *
+#  This file is used to set the parameters of the run.                 *
+#                                                                      *
+#  Some notation/conventions:                                          *
+#                                                                      *
+#   Lines starting with a hash (#) are info or comments                *
+#                                                                      *
+#   mind the format:   value    = variable     ! comment               *
+#***********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#
+#***********************************************************************
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of events (and their normalization) and the required          *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+  21000000 = nevents ! Number of unweighted events requested 
```

Comment:
@agrohsje This would pilot-run 2100K events right?

Reply:
:-D 
I was wondering how this gridpack was ever finished.

Reply:
Hello, the gridpack has been generated with few hundreds of events, I changed this number of events to point out here that we need around 20 M events. I did not know at which step I should request the number of events. I would be grateful if you let me know.

Reply:
2000-3000 should be okay (I think making this too small had some possibilities that it might be problematic, but not yet deeply investigated though, if I remember correctly)

The number of events you request for the samples would be something that you should send to PAG MC contacts.